File: rb/0408.rb
# 組合せ
def Comb(n, r)
    if (r == 0 || r == n)
        1
    elsif (r == 1)
        n
    else
        Comb(n - 1, r - 1) + Comb(n - 1, r)
    end
end

# 組合せ (異なる 10 個のものから 5 個取ってできる組合せの総数)
n = 10
r = 5
puts Comb(n, r)

File: py/0408.py
# coding: Shift_JIS

# 組合せ
def Comb(n, r):
    if (r == 0 or r == n):
        return 1
    elif (r == 1):
        return n
    else:
        return Comb(n - 1, r - 1) + Comb(n - 1, r)

# 組合せ (異なる 10 個のものから 5 個取ってできる組合せの総数)
n = 10
r = 5
print Comb(n, r)
